### DISCUSSION TITLE
Noob question

### DISCUSSION THREAD

**Discussion by jsalzy1** [jef/streetmerchant#1396] [Help]
I apologize for the stupid question probably. I only want to see when the 3060ti comes in and out of stock, so i put that it selected series but in the git all I see is the nvidia store and its going through the whole series. Thanks!

**Comment by FlyingToto** [comment]
you have to select the **SHOW_ONLY_SERIES=**    AND the stores(nvidia is the only one selected by default). **STORES=**

**Comment by jsalzy1** [comment]
I tried that like I put 3060ti so it should only show 3060ti dorm nvidia anyway so why does it show all. What is the file extension of that config file supposed to be?

**Reply by FlyingToto** [reply]
the file name is "dotenv" be careful on windows if you use notepad it will try to save it as "dotenv.txt" 
STORES=amazon,amd,antonline,walmart,asus,adorama,officedepot,target,bestbuy,bandh,newegg,nvidia
SHOW_ONLY_SERIES=3060ti
